packages/jokul/src/components/tooltip/documentation/TooltipExample.tsx

```diff
@@ -24,12 +26,17 @@ export const TooltipExample: FC<ExampleComponentProps> = ({ choiceValues }) => {
 
     return (
         <p>
-            Kontonummer:{" "}
+            Kontonummer: {formatKontonummer(kontonummer)}
             <Tooltip placement={initialPlacement} delay={delay}>
                 <TooltipTrigger onClick={copyToClipboard}>
-                    {formatKontonummer(kontonummer)}
+                    <Button
```

Comment:
Koden ser bra ut men jeg har et problem med eksempelet.

<img width="400" alt="image" src="https://github.com/user-attachments/assets/fbc95fdb-dea4-4dac-965a-d1a259e856be">

Det genereres en aria-label for triggeren som inkluderer `textContent` fra <TooltipTrigger>` som i dette tilfellet ender opp med å være noe gibberish i og med at vi bruker ikon-fonten. Akkurat hva fiksen på det blir er jeg litt usikker på men jeg syns vi burde finne en løsning på det.

Reply:
Ah, ja den bieffekten hadde jeg ikke tenkt på. Hvis vi har som regel at Tooltip skal brukes for knapper/handlinger som trenger forklaring trenger vi kanskje ikke lese opp `textContent`? 

Men er kanskje enklere å gå tilbake til det gamle eksemplet akkurat for denne endringen, selv om det er et litt dårlig eksempel.